### PR TITLE
1871 - IdsButton Test Conversion

### PR DESCRIPTION
**Pull request by n-ace-ancog**
**Explain the details for making this change. What existing problem does the pull request solve?**
Converted test scripts of `ids-button` component to Playwright.

**Related github/jira issue (required)**:
Closes enhancement https://github.com/infor-design/enterprise-wc/issues/1871

**Steps necessary to review your pull request (required)**:
- Test should pass
- No flaky result

**Included in this Pull Request**:
- [ ] Some documentation for the feature.
- [x] A test for the bug or feature.
- [ ] A note to the change log.

Updated to the latest branch and addresses review points from #1895.

In connection with the review points from https://github.com/infor-design/enterprise-wc/pull/1895#pullrequestreview-1901203168:

> a) Delete the tests from tests-to-convert that have been converted. **_DONE_**
b) Name the tests as if you were describing the scenario in it to a non-programmer familiar with the problem domain **_Renamed tests to be informative as possible to non-programmers**
c) ID could be id (super minor but its a bit mixed) **_DONE_**
d) Not sure what i think of so many test.steps since i never used it before? But at least if there is only one step we may not need it. Probably have to discuss. **_I've reduced the number of test.steps to lines which needs to be "grouped" together_**
e) dont need then because await is already doing this. **_DONE_**
f) also not sure why it looks like every single file changed in percy? **_Copied from a very outdated version of main hence the new pull request and branch_**

### CODE REVIEW

File: tests/ids-button/ids-button.spec.ts [hunk 1]
```diff
@@ -4,12 +4,14 @@ import { expect } from '@playwright/test';
...
 
 import IdsButton from '../../src/components/ids-button/ids-button';
+import { IdsButtonAppearance, IdsButtonIconAlignment, IdsButtonContentAlignment } from '../../src/components/ids-button/ids-button-common';
 
 test.describe('IdsButton tests', () => {
   const url = '/ids-button/example.html';
 
   test.beforeEach(async ({ page }) => {
     await page.goto(url);
+    await page.waitForLoadState();
```
Comment (tmcconechy) [inline tests/ids-button/ids-button.spec.ts:14]: Not sure if this is needed. Did you confirm? Might just take extra time

**Review by tmcconechy — COMMENTED** [review]
I think this looks good. Thanks @n-ace-ancog

File: tests/ids-button/ids-button.spec.ts (hunk 1 above)
Comment (n-ace-ancog) [reply tests/ids-button/ids-button.spec.ts:14]: You're right, seems like that the `goto` has already built in waiting for the `load` event to fire this makes the `waitForLoadState` redundant in this case. 

image

**Review by andrivarhanov — APPROVED** [review]

**Review by ericangeles — APPROVED** [review]
Looks good.

**Pull request merged**